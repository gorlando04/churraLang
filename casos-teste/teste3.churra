amigo "A" contribui 120
amigo "B" contribui 80 vegetariano
item carne "picanha" preco_kg 40 quantidade_kg 2
item comida preco_unit 10 quantidade 3
rateio igual