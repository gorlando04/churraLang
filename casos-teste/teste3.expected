Total do churrasco: R$110.00
A deve pagar: R$95.00
B deve pagar: R$15.00